ibm/data_source_ibm_private_dns_permitted_network.go

```diff
@@ -0,0 +1,155 @@
+package ibm
+
+import (
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	pdnsPermittedNetworks = "permitted_networks"
+)
+
+func dataSourceIBMPrivateDNSPermittedNetworks() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceIBMPrivateDNSPermittedNetworksRead,
+
+		Schema: map[string]*schema.Schema{
+
+			pdnsInstanceID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Instance ID",
+			},
+
+			pdnsZoneID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Zone Id",
+			},
+
+			pdnsPermittedNetworks: {
+
+				Type:        schema.TypeList,
+				Description: "Collection of permitted networks",
+				Computed:    true,
+				Elem: &schema.Resource{
+
+					Schema: map[string]*schema.Schema{
+
+						pdnsPermittedNetworkID: {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Network Id",
+						},
+
+						pdnsInstanceID: {
```

Comment:
Please Remove ForceNew, required ....since pdnsPermittedNetworks is computed so all sub arguments should also be declared as computed

Reply:
removed